jina/drivers/__init__.py

```diff
@@ -406,8 +406,8 @@ def __init__(self, traversal_paths: Tuple[str] = ('c', 'r'), *args, **kwargs):
         """Initialize a :class:`BaseRecursiveDriver`
 
         :param traversal_paths: Describes the leaves of the document tree on which _apply_all are called
-        :param *args: *args for super
-        :param **kwargs: **kwargs for super
+        :param args: additional positional arguments
```

Comment:
I would prefer we keep the information about how/where the *args and **kwargs are sent (what they are used for)

Reply:
thanks, done